examples/svid-mTLS/README.md

```diff
@@ -0,0 +1,86 @@
+# Mutual TLS example
+
+This example shows how two services can communicate using mTLS with X.509 SVIDs obtained from the SPIFFE workload API. 
+
+One of the workloads acts as a client and the other as the server. The first thing that both workloads do is setting the environment variable `SPIFFE_ENDPOINT_SOCKET` to the socket path of the SPIFFE provider. Since this example assumes the SPIRE implementation, it uses the SPIRE default socket path: `/tmp/agent.sock`. This value can also be set directly in the environment or using the `spiffe.WithWorkloadAPIAddr` function.
+
+Then, the **server workload** creates a listener using the `spiffe.ListenTLS` function.
+
+```go
+listener, err := spiffe.ListenTLS(context.Background(), "tcp", serverAddress, spiffe.ExpectPeer(clientSpiffeID))
+```
+
+On the other hand, the **client workload** creates a `net.Conn` object using the `spiffe.DialTLS` function.
+
+```go
+conn, err := spiffe.DialTLS(ctx, "tcp", serverAddress, spiffe.ExpectPeer(serverSpiffeID))
+```
+
+In both cases, a `spiffe.ExpectPeerFunc` is set to validate the workload is authorized to connect to the other peer. In this example, the `spiffe.ExpectPeer` was used to allow the client to reach the server and vice versa.
+
+That is it. The go-spiffe library fetches and automatically renews the X.509 SVIDs of both workloads according to the SPIFFE policy.
```

Comment:
how often are the SVIDs renewed ? where is it defined the TTL ?

Reply:
That is defined in the Workload API provider configuration. In this example, it is defined in the SPIRE server configuration file. I'll clarify it.